### DISCUSSION TITLE
Welcome to r-inla Discussions!

### DISCUSSION THREAD

**Discussion by hrue** [hrue/r-inla#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mYythc** [comment]
hello! I just cannot load www.r-inla.org(because of some network limitation) and downloaded codes about r-inla created by hrue. could you tell me what to do next with so many files?

**Comment by hrue** [comment]
can you download directly from 

https://inla.r-inla-download.org/R/testing/

?

On Wed, 2022-08-17 at 00:13 -0700, mYythc wrote:
> hello! I just cannot load www.r-inla.org(because of some network
> limitation) and downloaded codes about r-inla created by hrue. could
> you tell me what to do next with so many files?
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>

-- 
Håvard Rue
Professor of Statistics
Statistics Program
CEMSE Division
King Abdullah University of Science and Technology
Thuwal 23955-6900
Kingdom of Saudi Arabia

***@***.***
Office: +966 (0)12 808 0640  
Mobile: +966 (0)54 470 0421
Research group: bayescomp.kaust.edu.sa  
R-INLA project: www.r-inla.org
Zoom: kaust.zoom.us/my/haavard.rue
Zoom.office: Meeting ID 510 522 9264

**Reply by mYythc** [reply]
> i can load the website and see things like this:
  | Name | Last modified | Size | Description
  | Parent Directory |   | - |  
  | stable/ | 2015-01-02 17:56 | - |  
  | testing/ | 2015-01-02 17:56 | - |  

> i just clicked each of them:
Linux-builds/ | 2022-07-23 00:35 | - |  
  | R/ | 2015-01-02 22:00 | - |  
  | givemeINLA-testing.R | 2022-07-23 00:32 | 4.4K |  
  | givemeINLA.R | 2022-07-23 00:32 | 4.4K |  
  | logfile-12705 | 2022-07-21 08:38 | 29K |  
  | logfile-236009 | 2022-06-30 13:44 | 29K |  
  | logfile-392501 | 2022-07-21 11:34 | 29K |  
  | logfile-466969 | 2022-07-23 01:05 | 22K |  
  | make-md5 | 2021-03-31 14:35 | 430 |  
  | r-inla.org/ | 2020-06-30 16:20 | - |  
  | robots.txt | 2019-11-02 10:01 | 31 |  
  | updatePackageIndex | 2022-07-21 11:30 | 303 |  
  | validate | 2020-07-18 15:41 | 664 |  

> how to download them? could you tell me what to do next? thanks a lot!

**Comment by hrue** [comment]
then there should not be any issue with doing

options(timeout=9999)
install.packages("INLA",repos=c(getOption("repos"),INLA="https://inla.r-inla-download.org/R/testing
"), dep=TRUE)





On Wed, 2022-08-17 at 01:00 -0700, mYythc wrote:
> > i can load the website and see things like this:
> >   | Name | Last modified | Size | Description
> >   | Parent Directory |   | - |  
> >   | stable/ | 2015-01-02 17:56 | - |  
> >   | testing/ | 2015-01-02 17:56 | - |  
> > i just clicked each of them:
> > Linux-builds/ | 2022-07-23 00:35 | - |  
> >   | R/ | 2015-01-02 22:00 | - |  
> >   | givemeINLA-testing.R | 2022-07-23 00:32 | 4.4K |  
> >   | givemeINLA.R | 2022-07-23 00:32 | 4.4K |  
> >   | logfile-12705 | 2022-07-21 08:38 | 29K |  
> >   | logfile-236009 | 2022-06-30 13:44 | 29K |  
> >   | logfile-392501 | 2022-07-21 11:34 | 29K |  
> >   | logfile-466969 | 2022-07-23 01:05 | 22K |  
> >   | make-md5 | 2021-03-31 14:35 | 430 |  
> >   | r-inla.org/ | 2020-06-30 16:20 | - |  
> >   | robots.txt | 2019-11-02 10:01 | 31 |  
> >   | updatePackageIndex | 2022-07-21 11:30 | 303 |  
> >   | validate | 2020-07-18 15:41 | 664 |  
> > how to download them? could you tell me what to do next? thanks a
> > lot!
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>

-- 
Håvard Rue
Professor of Statistics
Statistics Program
CEMSE Division
King Abdullah University of Science and Technology
Thuwal 23955-6900
Kingdom of Saudi Arabia

***@***.***
Office: +966 (0)12 808 0640  
Mobile: +966 (0)54 470 0421
Research group: bayescomp.kaust.edu.sa  
R-INLA project: www.r-inla.org
Zoom: kaust.zoom.us/my/haavard.rue
Zoom.office: Meeting ID 510 522 9264

**Reply by mYythc** [reply]
something sad:(
Warning in install.packages :
  unable to access index for repository https://inla.r-inla-download.org/R/testing 
/src/contrib:
  cannot open URL 'https://inla.r-inla-download.org/R/testing 
/src/contrib/PACKAGES'
Warning in install.packages :
  package ‘INLA’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
Warning in install.packages :
  unable to access index for repository https://inla.r-inla-download.org/R/testing 
/bin/windows/contrib/4.2:
  cannot open URL 'https://inla.r-inla-download.org/R/testing 
/bin/windows/contrib/4.2/PACKAGES'

i can load the website but why cannot open URL???

**Comment by hrue** [comment]
you need R-4.2


On Wed, 2022-08-17 at 01:31 -0700, mYythc wrote:
> something sad:(
> Warning in install.packages :
> unable to access index for repository
> https://inla.r-inla-download.org/R/testing
> /src/contrib:
> cannot open URL 'https://inla.r-inla-download.org/R/testing
> /src/contrib/PACKAGES'
> Warning in install.packages :
> package ‘INLA’ is not available for this version of R
> A version of this package for your version of R might be available
> elsewhere,
> see the ideas at
> https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
> Warning in install.packages :
> unable to access index for repository
> https://inla.r-inla-download.org/R/testing
> /bin/windows/contrib/4.2:
> cannot open URL 'https://inla.r-inla-download.org/R/testing
> /bin/windows/contrib/4.2/PACKAGES'
> i can load the website but why cannot open URL???
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>

-- 
Håvard Rue
Professor of Statistics
Statistics Program
CEMSE Division
King Abdullah University of Science and Technology
Thuwal 23955-6900
Kingdom of Saudi Arabia

***@***.***
Office: +966 (0)12 808 0640  
Mobile: +966 (0)54 470 0421
Research group: bayescomp.kaust.edu.sa  
R-INLA project: www.r-inla.org
Zoom: kaust.zoom.us/my/haavard.rue
Zoom.office: Meeting ID 510 522 9264

**Reply by mYythc** [reply]
when i input _library()_ 
i can see INLA in R package available 
and the version is 22.05.07
Does this prove that I downloaded successfully??!!

**Comment by hrue** [comment]
try download the recent testing version and make sure you're using R-4.2

On Wed, 2022-08-17 at 04:09 -0700, mYythc wrote:
> when i input library()
> i can see INLA in R package available
> and the version is 22.05.07
> Does this prove that I downloaded successfully??!!
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>

-- 
Håvard Rue
Professor of Statistics
Statistics Program
CEMSE Division
King Abdullah University of Science and Technology
Thuwal 23955-6900
Kingdom of Saudi Arabia

***@***.***
Office: +966 (0)12 808 0640  
Mobile: +966 (0)54 470 0421
Research group: bayescomp.kaust.edu.sa  
R-INLA project: www.r-inla.org
Zoom: kaust.zoom.us/my/haavard.rue
Zoom.office: Meeting ID 510 522 9264

**Comment by Wenhuan2516** [comment]
Hi,

I keep getting this error: Error: inla.mkl: src/my.c:172: my_gsl_sf_lnchoose_e: Assertion `0 == 1' failed.
inla.mkl: src/my.c:172: my_gsl_sf_lnchoose_e: Assertion `0 == 1' failed.
sh: line 1: 860218 Aborted                 '/global/u1/w/wenhuan/.conda/envs/r-inla-env/lib/R/library/INLA/bin/linux/64bit/inla.mkl.run' -s -v -t256:1 -B0 -P compact '/tmp/RtmpIOSdd3/filec3223208b0334/Model.ini' > '/tmp/RtmpIOSdd3/filec3223208b0334/Logfile.txt'
Error in inla.inlaprogram.has.crashed() :
  The inla-program exited with an error. Unless you interupted it yourself, please rerun with verbose=TRUE and check the output carefully.
  If this does not help, please contact the developers at <[help@r-inla.org](mailto:help@r-inla.org)>.
Error in inla.core.safe(formula = formula, family = family, contrasts = contrasts,  :
  *** Fail to get good enough initial values. Maybe it is due to something else. How to fix it?

**Reply by hrue** [reply]
I guess there is something wrong in the input. 


	if (m > n) {
		assert(0 == 1);

so you're asking for binomial, f.ex where y=6 and Ntrials=2
